docs/3.api/2.composables/use-async-data.md

```diff
@@ -61,6 +61,7 @@ const { data: posts } = await useAsyncData(
   - `immediate`: when set to `false`, will prevent the request from firing immediately. (defaults to `true`)
   - `default`: a factory function to set the default value of the `data`, before the async function resolves - useful with the `lazy: true` or `immediate: false` option
   - `transform`: a function that can be used to alter `handler` function result after resolving
+  - `useCachedData`: a function which returns cached data. A *nullish* return will trigger a fetch.
```

Comment:
Might be good to mention the default somewhere ☺️

Reply:
The default? As in, returning data in this function returns that as the data?

Reply:
Please see what you think of the updated wording 👍